systemservices/sources/workflow/workflow/proto/api.pb.go

```diff
@@ -0,0 +1,2817 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: github.com/mesg-foundation/core/protobuf/coreapi/api.proto
+
+package coreapi
```

Comment:
It's because of we vendor all packages inside each systemservice. We can use one vendor per core and use it (in such case each service will be importing from main vendor)

Reply:
Hum I see...
This API is not supposed to be accessible, a service should not access the host api, even if for now it's technically possible this should be dropped this can be a huge risk and we can always use the service composition for that.
This part should be delegated to the core, I really see the following process
- The core receive an event from a service
- The core ask the workflow system service the matching tasks 
- The core execute the tasks

Reply:
I don't know why it's a risk to use public APIs of core from a service. I need a much detailed overview of service composition to have more opinions. Maybe we can keep discussion moving forward on the related forum proposal. 

If we listen events & results and execute tasks on core itself, that means we're going to move a good amount of logic from WSS. The main goal was to split core to system services and move logic from core to system services as much as possible.

I'd like to hear more to get a better understanding of your view.

Reply:
we discussed about this one on the weekly meeting and we decided that core itself will be responsible for listening events & results and executing tasks. 

so we'll have two new tasks on the WSS to lookup for what tasks should be executed depending on results or events. 

proposed in the chat:
```
task: lookupTaskFromEvent -> output with array of tasks with their input data that needed to be executed
task: lookupTaskFromResult -> output with array of tasks with their input data that needed to be executed
```